BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx

```diff
@@ -0,0 +1,163 @@
+import React, {Component, PropTypes} from 'react';
+import {isEmpty, bindAll} from 'underscore';
+import {Link} from 'react-router';
+import Alert from 'react-bootstrap/lib/Alert';
+import {contains} from 'underscore';
+import classNames from 'classnames';
+
+const initialState = {
+  expanded: false
+};
+
+class InterProjectAlert extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = initialState;
```

Comment:
this was a common mistake that people were making in backbone where all instances of a view are referencing the same initial/default object. it probably isn't a big problem since I think all calls to setState sets this.state to a completely new object, but it might lead to subtle bugs. Most of the examples that I've seen online seem to create a new object in the constructor
```
constructor(props) {
  super(props)
  this.state = {
    expanded: false
  }
}